diff -burN NewsPortal-0.36.orig/article-raw.php NewsPortal-0.36/article-raw.php
--- NewsPortal-0.36.orig/article-raw.php	2005-01-10 10:57:07.812216672 +0100
+++ NewsPortal-0.36/article-raw.php	2005-01-10 10:57:44.792594800 +0100
@@ -1,6 +1,6 @@
 <?
   header("Expires: ".gmdate("D, d M Y H:i:s",time()+(3600*24))." GMT");
-  include "config.inc.php";
+  include "/etc/webapps/newsportal/config.php";
     $attachment_delete_alternative=false;
     $attachment_uudecode=false;
 
diff -burN NewsPortal-0.36.orig/article.php NewsPortal-0.36/article.php
--- NewsPortal-0.36.orig/article.php	2005-01-10 10:57:07.813216520 +0100
+++ NewsPortal-0.36/article.php	2005-01-10 10:57:58.445519240 +0100
@@ -5,7 +5,7 @@
   $id=$_REQUEST["id"];
   $group=$_REQUEST["group"];
 
-  include "config.inc.php";
+  include "/etc/webapps/newsportal/config.php";
   include "auth.inc";
   include "$file_newsportal";
 
diff -burN NewsPortal-0.36.orig/attachment.php NewsPortal-0.36/attachment.php
--- NewsPortal-0.36.orig/attachment.php	2005-01-10 10:57:07.813216520 +0100
+++ NewsPortal-0.36/attachment.php	2005-01-10 10:58:10.155739016 +0100
@@ -4,7 +4,7 @@
 $id=$_REQUEST["id"];
 $attachment=$_REQUEST["attachment"];
 
-include "config.inc.php";
+include "/etc/webapps/newsportal/config.php";
 include "auth.inc";
 require("$file_newsportal");
 if (!isset($attachment))
diff -burN NewsPortal-0.36.orig/config.inc.php NewsPortal-0.36/config.inc.php
--- NewsPortal-0.36.orig/config.inc.php	2005-01-10 10:57:07.812216672 +0100
+++ NewsPortal-0.36/config.inc.php	2005-01-10 10:59:41.505851688 +0100
@@ -2,7 +2,7 @@
 /*
  * directories and files
  */
-$spooldir="spool";
+$spooldir="/var/spool/newsportal";
 $imgdir="img";
 $file_newsportal="newsportal.php";
 $file_index="index.php";
@@ -14,7 +14,7 @@
 $file_cancel="cancel.php";
 $file_language="lang/english.lang";
 $file_footer="";
-$file_groups="groups.txt";
+$file_groups="/etc/webapps/newsportal/groups.txt";
 
 /* 
  * newsserver setup
diff -burN NewsPortal-0.36.orig/index.php NewsPortal-0.36/index.php
--- NewsPortal-0.36.orig/index.php	2005-01-10 10:57:07.813216520 +0100
+++ NewsPortal-0.36/index.php	2005-01-10 10:58:46.264249688 +0100
@@ -1,6 +1,6 @@
 <? header("Expires: ".gmdate("D, d M Y H:i:s",time()+7200)." GMT");
 
-   include "config.inc.php";
+   include "/etc/webapps/newsportal/config.php";
    include "auth.inc";
    include "head.inc"; ?>
 
diff -burN NewsPortal-0.36.orig/post.php NewsPortal-0.36/post.php
--- NewsPortal-0.36.orig/post.php	2005-01-10 10:57:07.814216368 +0100
+++ NewsPortal-0.36/post.php	2005-01-10 10:59:17.577489352 +0100
@@ -34,7 +34,7 @@
 @$id=$_REQUEST["id"];
 if (!isset($group)) $group=$newsgroups;
 
-include "config.inc.php";
+include "/etc/webapps/newsportal/config.php";
 include "auth.inc";
 
 // Save name and email in cookies
diff -burN NewsPortal-0.36.orig/thread.php NewsPortal-0.36/thread.php
--- NewsPortal-0.36.orig/thread.php	2005-01-10 10:57:07.812216672 +0100
+++ NewsPortal-0.36/thread.php	2005-01-10 10:59:28.401843800 +0100
@@ -7,7 +7,7 @@
   if(isset($_REQUEST["last"]))
     $last=intval($_REQUEST["last"]);
 
-  include "config.inc.php";
+  include "/etc/webapps/newsportal/config.php";
 
   include "auth.inc";
   $title.= ' - '.$group;
