diff -burNp NewsPortal-0.38/article.php NewsPortal-0.38-new/article.php
--- NewsPortal-0.38/article.php	2006-10-08 12:23:18.000000000 +0200
+++ NewsPortal-0.38-new/article.php	2009-01-15 11:51:58.634806113 +0100
@@ -6,7 +6,7 @@
   $id=$_REQUEST["id"];
   $group=$_REQUEST["group"];
 
-  include "config.inc.php";
+  include "/etc/webapps/newsportal/config.php";
 
   /*
   $thread_show["replies"]=true;
diff -burNp NewsPortal-0.38/attachment.php NewsPortal-0.38-new/attachment.php
--- NewsPortal-0.38/attachment.php	2006-10-08 12:23:18.000000000 +0200
+++ NewsPortal-0.38-new/attachment.php	2009-01-15 11:52:15.838554484 +0100
@@ -4,7 +4,7 @@ $group=$_REQUEST["group"];
 $id=$_REQUEST["id"];
 $attachment=$_REQUEST["attachment"];
 
-include "config.inc.php";
+include "/etc/webapps/newsportal/config.php";
 include "auth.inc";
 require("$file_newsportal");
 if (!isset($attachment))
diff -burNp NewsPortal-0.38/config.inc.php NewsPortal-0.38-new/config.inc.php
--- NewsPortal-0.38/config.inc.php	2008-07-16 12:18:46.000000000 +0200
+++ NewsPortal-0.38-new/config.inc.php	2009-01-15 11:53:18.234801158 +0100
@@ -2,7 +2,7 @@
 /*
  * directories and files
  */
-$spooldir="spool";
+$spooldir="/var/spool/newsportal";
 $imgdir="img";
 $file_newsportal="newsportal.php";
 $file_index="index.php";
@@ -14,7 +14,7 @@ $file_post="post.php";
 $file_cancel="cancel.php";
 $file_language="lang/english.lang";
 $file_footer="";
-$file_groups="groups.txt";
+$file_groups="/etc/webapps/newsportal/groups.txt";
 
 /* 
  * newsserver setup
diff -burNp NewsPortal-0.38/index.php NewsPortal-0.38-new/index.php
--- NewsPortal-0.38/index.php	2006-10-08 12:23:18.000000000 +0200
+++ NewsPortal-0.38-new/index.php	2009-01-15 11:54:11.311458999 +0100
@@ -1,6 +1,6 @@
 <? header("Expires: ".gmdate("D, d M Y H:i:s",time()+7200)." GMT");
 
-   include "config.inc.php";
+   include "/etc/webapps/newsportal/config.php";
    include "auth.inc";
    include "head.inc"; ?>
 
diff -burNp NewsPortal-0.38/post.php NewsPortal-0.38-new/post.php
--- NewsPortal-0.38/post.php	2008-07-16 11:58:48.000000000 +0200
+++ NewsPortal-0.38-new/post.php	2009-01-15 11:54:39.371456502 +0100
@@ -20,7 +20,7 @@
  *  along with this program; if not, write to the Free Software
  *  Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
  */
-include "config.inc.php";
+include "/etc/webapps/newsportal/config.php";
 
 @$fieldnamedecrypt=$_REQUEST['fielddecrypt'];
 //@$newsgroups=$_REQUEST["newsgroups"];
diff -burNp NewsPortal-0.38/thread.php NewsPortal-0.38-new/thread.php
--- NewsPortal-0.38/thread.php	2008-07-16 11:50:44.000000000 +0200
+++ NewsPortal-0.38-new/thread.php	2009-01-15 11:55:08.748536889 +0100
@@ -8,7 +8,7 @@ if(isset($_REQUEST["first"]))
 if(isset($_REQUEST["last"]))
   $last=intval($_REQUEST["last"]);
 
-include "config.inc.php";
+include "/etc/webapps/newsportal/config.php";
 include("$file_newsportal");
 include "auth.inc";
 $title.= ' - '.$group;
